server/data/data-jmap/src/main/java/org/apache/james/jmap/api/preview/Preview.java

```diff
@@ -20,18 +20,35 @@
 package org.apache.james.jmap.api.preview;
 
 import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Preconditions;
 
 public class Preview {
 
     private static final int MAX_LENGTH = 256;
+    public static final Preview NO_BODY = new Preview("(Empty)");
```

Comment:
You didn't answered that question: https://github.com/linagora/james-project/pull/2918#discussion_r350219129

Reply:
by bad, I use it in the `MessageFullViewFactory` when textBody is empty:
```java
        Optional<String> mainTextContent = mainTextContent(messageContent);
        Preview preview = mainTextContent.map(Preview::compute)
            .orElse(Preview.NO_BODY);
```

Reply:
I bet it's a hardcoded - locale unfriendly constant.

It displays as `(Empty)` despite my french locale.

I believe we should open a bug report. However we should synchronize with OpenPaaS to solve that issue.

Will handle that as a separated issue.

Reply:
I created the OpenPaaS tickets...

We will provide a separated PR for that breaking change behaviour.

Reply:
We have to define a Null Object for this case and interpret it in the presentation layer.
It can/must be done right now, we'll take care of the locale-friendlyness later but at least we would not pass an hardcoded string around.

Reply:
Thank you for pointing out that spot!